packages/braid-design-system/src/lib/components/Notice/Notice.docs.tsx

```diff
@@ -47,16 +53,35 @@ const docs: ComponentDocs = {
   alternatives: [
     {
       name: 'Alert',
-      description: 'For a stronger visual treatment.',
+      description:
+        'For page and section level messages, or for a stronger visual treatment.',
     },
     {
       name: 'useToast',
-      description: 'For asynchronous messages that float above the page.',
+      description:
+        'To acknowledges a user action without interrupting their flow.',
     },
   ],
   additional: [
     {
-      label: 'Content guidelines',
+      label: 'Choosing a tone',
+      description: (
+        <Stack space="large">
+          <Text>
+            Use the <Strong>tone</Strong> property to help communicate the
+            meaning behind the Notice. Notices support{' '}
+            <Strong>promote, info, positive</Strong> and{' '}
+            <Strong>critcal</Strong> tones.
+          </Text>
+          <Text>
+            Read more about{' '}
+            <TextLink href="/foundations/tones">Braid tones</TextLink>.
```

Comment:
```suggestion
            <TextLink href="/foundations/tones">Tones</TextLink>.
```
Possibly can drop 'Braid' here to keep in line with the other links?

Reply:
Good call. I'll update in Alert.docs.tsx as well 👍